PHP/Language.md

```diff
@@ -154,29 +174,38 @@ $this vs self
 
 static properties and methods
 
-Magic methods (constructors, object as callable, overloading etc)
+Magic methods 
+* constructors, 
+* overloading methods and properties
+* object as callable
+* serialization and json encoding control
+     __serialize/__unserialize. vs __sleep/__wakeup
```

Comment:
__serialize/__unserialize are not magic methods, right?

Reply:
They are in php7.4. https://wiki.php.net/rfc/custom_object_serialization